app/templates/components/layer-record-views/zoning-map-index.hbs

```diff
@@ -0,0 +1,56 @@
+<label class="header-type">
+  ZONING MAP INDEX
+</label>
+<h1 class="header-large" style="text-transform: uppercase;">
+  {{if (eq this.model.id "30d-Subplan") "SUB PLAN OF 30D" this.model.id}}
+</h1>
+
+<p style="margin-bottom: 0.25rem;">
```

Comment:
Can you move these into a scss file under `/styles/modules`? Inline styling tends to cause headaches down the road.